packages/story-editor/src/components/floatingMenu/elements/borderWidthAndColor.js

```diff
@@ -137,8 +137,10 @@ function BorderWidthAndColor() {
           value={border.color || BLACK}
           onChange={handleColorChange}
           hasInputs={false}
-          hasEyeDropper={false}
+          hasEyedropper={false}
           allowsOpacity={canHaveBorderOpacity}
```

Comment:
I think this is gonna need to be false as well 🤔 Since I can go to custom colors, it will show gradient as an option then 💥 
<img width="1344" 
<img width="352" alt="Screen Shot 2022-03-22 at 2 50 28 PM" src="https://user-images.githubusercontent.com/1820266/159573552-6e09ee36-16a0-477d-9af8-094a95b370c2.png">
alt="Screen Shot 2022-03-22 at 2 48 11 PM" src="https://user-images.githubusercontent.com/1820266/159573392-3ddc96fc-e4a8-4423-b846-0244f4c6863f.png">

Reply:
Or rather, `allowsGradient` should be false.

Reply:
oh yup, you're super right. i completely ignored that button.